src/improvements/tasks/MultisigTask.sol

```diff
@@ -901,6 +905,12 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager, TaskManage
     /// State overrides are not applied yet. Keep this in mind when performing various pre-simulation assertions in this function.
     function _templateSetup(string memory taskConfigFilePath, address rootSafe) internal virtual;
 
+    /// @notice Sets the allowed storage accesses.
```

Comment:
These were introduced ultimately to support being able to set a variable in _templateSetup and then use that variable in `_taskStorageWrites()`. See the new ordering of the functions above and the `SetEIP1967Impl` template for an example of this in action.